addons/sourcemod/scripting/influx_speclist.sp

```diff
@@ -108,30 +108,30 @@ public Action Cmd_SpecList( int client, int args )
         {
             ++num;
             
-            Format( szMsg, sizeof( szMsg ), "%s%s{MAINCLR1}%N", szMsg, ( szMsg[0] ) ? "{CHATCLR}, " : "", i );
+            Format( szMsg, sizeof( szMsg ), "%s\x01\x04%N", szMsg, i );
```

Comment:
All hardcoded colors should just use the translation files. There needs to be a way to change the colors somewhere.

Reply:
Ok, I will add an item separator to `influx_palette.ini`